charts/flyte-core/templates/propeller/crds/flyteworkflow.yaml

```diff
@@ -1,5 +1,9 @@
 {{- if .Values.flytepropeller.enabled }}
+{{- if $.Capabilities.APIVersions.Has "apiextensions.k8s.io/v1/CustomResourceDefinition" }}
```

Comment:
Do people still use Flyte with k8s versions `<1.16`? If not, this might not be necessary.